CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog
 =========
 
+3.14.0 (TBD)
+-------------------
+
+* Reset the random state for Model Bakery.
```

Comment:
Let's link, it could be useful for those who aren't sure what it is.

```suggestion
* Reset the random state for `Model Bakery <https://model-bakery.readthedocs.io/en/latest/>`__.
```